transformers/modeling_utils.py

```diff
@@ -479,6 +479,11 @@ def load(module, prefix=''):
             loading_info = {"missing_keys": missing_keys, "unexpected_keys": unexpected_keys, "error_msgs": error_msgs}
             return model, loading_info
 
+        if xla_device:
+            import torch_xla.core.xla_model as xm
+            model = xm.send_cpu_data_to_device(model, xm.xla_device())
+            model = model.to(xm.xla_device())
```

Comment:
are these doing the same thing?

Reply:
Not quite. Based on what I've tested, the first doesn't set the tensors to device `xla`. Is was only after the second that it went to device.

Reply:
oh ok the first one is for sending auxiliary tensors to the device, whereas the second one sends model params to device.